debian_on_termux.sh

```diff
@@ -9,7 +9,7 @@ DO_FIRST_STAGE=: # false   # required (unpack phase/ executes outside guest invi
 DO_SECOND_STAGE=: # false  # required (complete the install/ executes inside guest invironment)
 DO_THIRD_STAGE=: # false   # optional (enable local policies/ executes inside guest invironment)
 
-ARCHITECTURE=$(uname -m | sed 's/aarch64/arm64/g')
+ARCHITECTURE=$(uname -m | sed 's/aarch64/arm64/g' | sed 's/x86_64/amd64/g')
```

Comment:
Can we use 
's/aarch64/arm64/g;s/x86_64/amd64/g'
?

Reply:
Oh yeah, agreed